codegen-luau/runtime/runtime.lua

```diff
@@ -85,6 +87,20 @@ do
 		return -num
 	end
 
+	function min.num(a, b)
```

Comment:
It's unclear whether Luau behavior here is a bug or intended, so this may or may not be needed. If it is, `math.min` and `math.max` should be localized, and a new line after the if statement should be present.